Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Spawnable/AssetPlatformComponentRemover.cpp

```diff
@@ -54,8 +54,8 @@ namespace AzToolsFramework::Prefab::PrefabConversionUtils
                     {
                         // Loop over an entity's components backwards and pop-off components that shouldn't exist.
                         AZStd::vector<AZ::Component*> components = entity->GetComponents();
-                        const int oldComponentCount = components.size();
-                        for (int i = oldComponentCount - 1; i >= 0; --i)
+                        const auto oldComponentCount = components.size();
```

Comment:
Minor: could have casted to int on this line, as was done with the other change, instead of using auto

Reply:
True, but I thought it was better to keep the count a size_t for clarity.